week-2-practise/level-1/4-slice.js

```diff
@@ -5,7 +5,7 @@
   DO NOT EDIT BELOW THIS LINE
   --------------------------- */
 var numbers = [1, 2, 3, 4, 5, , 6, 7, 8];
-var first5Numbers = first5(numbers);
+var first5Numbers = numbers.slice(0,5);
```

Comment:
You could do it this way, but the instructions specifically wanted you to create a function, and it's more readable that way too :-)